src/components/EmailValidationSection.tsx

```diff
@@ -0,0 +1,30 @@
+import * as React from "react";
+import { Store } from "redux";
+import { State } from "../reducers/index";
+import { LibraryData } from "../interfaces";
+import EmailValidationForm from "./EmailValidationForm";
+
+export interface EmailValidationSectionProps {
+  library: LibraryData;
+  store: Store<State>;
+}
+
+export default class EmailValidationSection extends React.Component<EmailValidationSectionProps, {}> {
+  render(): JSX.Element {
+    let emailForms = ["contact", "help", "copyright"].map((x) =>
```

Comment:
A comment around the context for why these three email values are being validated would be nice for developers.

Reply:
@adriana-alter

Reply:
And again.  Github being weird.

<img width="1179" alt="Screen Shot 2020-03-09 at 5 05 52 PM" src="https://user-images.githubusercontent.com/42178216/76257544-412e7900-6228-11ea-9f99-902b61ffa82a.png">